Tasks/VsTest/vstest.ts

```diff
@@ -493,6 +498,11 @@ function getVstestTestsList(vsVersion: number): Q.Promise<string> {
     }
 
     let vstest = tl.tool(vsVersionDetails.location);
+
+    if(vsVersion === 14) {
+        let vsTestPath = path.join(__dirname, "TestSelector/14.0/vstest.console.exe") // Use private vstest as the changes to discover tests are not there in update3
```

Comment:
We will use probing path for assembly redirections? Have we tested this? Remember facing some issue last time while trying this.